locale/fr.po

```diff
@@ -0,0 +1,1599 @@
+
+#: lib/utils/pyexec.c:97 py/builtinimport.c:253
+msgid "script compilation not supported"
+msgstr "compilation du script non supporté"
+
+#: main.c:137
+msgid " output:\n"
+msgstr " sortie:\n"
+
+#: main.c:148 main.c:221
+msgid ""
+"Auto-reload is on. Simply save files over USB to run them or enter REPL to "
+"disable.\n"
+msgstr ""
+"Auto-chargement activé. Copiez simplement les fichiers en USB pour les"
+"lancer ou entrez sur REPL pour le désactiver.\n"
+
+#: main.c:150
+msgid "Running in safe mode! Auto-reload is off.\n"
+msgstr "Mode sans-échec. Auto-rechargement désactivé.\n"
+
+#: main.c:152 main.c:223
+msgid "Auto-reload is off.\n"
+msgstr "Auto-rechargement désactivé.\n"
+
+#: main.c:166
+msgid "Running in safe mode! Not running saved code.\n"
+msgstr "Mode sans-échec! Le code sauvegardé ne tourne pas.\n"
+
+#: main.c:182
+msgid "WARNING: Your code filename has two extensions\n"
+msgstr "ATTENTION: le nom de fichier de votre code a deux extensions\n"
+
+#: main.c:230
+msgid "You requested starting safe mode by "
+msgstr "Vous avez demandé à démarrer en mode sans-échec par "
+
+#: main.c:233
+msgid "To exit, please reset the board without "
+msgstr "Pour quitter, redémarrez la carte SVP sans "
+
+#: main.c:240
+msgid ""
+"You are running in safe mode which means something really bad happened.\n"
+msgstr "Vous êtes en mode sans-échec ce qui signifie que quelque chose de"
+"mauvais est arrivé.\n"
+
+#: main.c:242
+msgid "Looks like our core CircuitPython code crashed hard. Whoops!\n"
+msgstr "Il semblerait que votre code CircuitPython a durement planté. Oups!\n"
+
+#: main.c:243
+msgid "Please file an issue here with the contents of your CIRCUITPY drive:\n"
+msgstr "SVP, remontez le problème là avec le contenu du lecteur CIRCUITPY:\n"
+
+#: main.c:246
+msgid ""
+"The microcontroller's power dipped. Please make sure your power supply "
+"provides\n"
+msgstr ""
+"L'alimentation du microcontroleur a chuté. Merci de vérifier que votre "
+"alimentation fournit\n"
+
+#: main.c:247
+msgid ""
+"enough power for the whole circuit and press reset (after ejecting "
+"CIRCUITPY).\n"
+msgstr ""
+"assez de puissance pour l'ensemble du circuit et appuyez sur 'reset' "
+"(après avoir éjecter CIRCUITPY).\n"
+
+#: main.c:251
+msgid "Press any key to enter the REPL. Use CTRL-D to reload."
+msgstr "Appuyez sur une touche pour entrer sur REPL ou CTRL-D pour recharger."
+
+#: main.c:406
+msgid "soft reboot\n"
+msgstr "redémarrage logiciel\n"
+
+#: ports/atmel-samd/common-hal/analogio/AnalogIn.c:63
+msgid "Pin does not have ADC capabilities"
+msgstr "la broche ne peut être utilisé pour l'ADC"
+
+#: ports/atmel-samd/common-hal/analogio/AnalogOut.c:53
+msgid "AnalogOut not supported on given pin"
+msgstr "AnalogOut n'est pas supporté sur cette broche"
+
+#: ports/atmel-samd/common-hal/audiobusio/I2SOut.c:147
+#: ports/atmel-samd/common-hal/audiobusio/I2SOut.c:150
+msgid "Invalid bit clock pin"
+msgstr "Broche invalide pour le bit d'horloge"
```

Comment:
'bit d'horloge' seems strange : it's the clock used to measure the time of one bit in the stream, so more like 'horloge des bits'. But it maybe easier to use 'bit clock', as it's the name of the signal.

Reply:
I changed line #120 too for consistency:
```
msgstr "'bit clock' et 'word select' doivent partagé une horloge"
````